.github/workflows/deploy-to-gh-pages.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Deploy Webpage
+on: 
+  push:
+    branches:
+      performance-benchmark-page
+
+permissions:
+  contents: write
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install and Build
+        run: |
+          npm install -f 
+          npm run build
+        env:
+          REACT_APP_GATEWAY: ${{ secrets.REACT_APP_GATEWAY }}
+          REACT_APP_GATEWAY_API_KEY: ${{ secrets.REACT_APP_GATEWAY_API_KEY }}
+
+
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
Is there not a GitHub approved action for this? I'm a little concerned about adding this.

Reply:
Github even uses this action as an example here. Not sure if that counts as official.
https://github.com/marketplace/actions/deploy-to-github-pages

Reply:
Looking for alternatives now.